acceptance_tests/features/export_file.feature

```diff
@@ -10,16 +10,16 @@ Feature: Export files can be created and sent with correct data
 
     Examples:
       | sample file                      | template                                               |
-      | social_sample_3_lines_fields.csv | ["ADDRESS_LINE1","ADDRESS_LINE2","POSTCODE","__uac__"] |
+      | social_sample_3_lines_fields.csv | address_line1__address_line2__postcode__uac |
 
     @regression
     Examples:
       | sample file                 | template                                                     |
-      | business_sample_6_lines.csv | ["BUSINESS_NAME","TOWN_NAME","__uac__","__qid__","INDUSTRY"] |
+      | business_sample_6_lines.csv | business_name__town_name__uac__qid__industry|
```

Comment:
PyCharm can auto-format these tables to keep them neat (and avoid unnecessary future formatting changes).